### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by aalexgabi** [pedromsilvapt/composable#1]
First I would like to thank you for this library. It's very useful and saves a lot of time.

I found some errors when I run the example (modified for common js):
```js
const { trim, concat, source } = require("composable");
const { color, silence } = require("composable");
const { separator } = require("composable");
const { compile, command } = require("composable");
 
// These will be the times we want to trim
const times = [ [ 0, 10 ], [ 20, 30 ], [ 40, 50 ] ];
 
const video = [];
const audio = [];
 
let file = source( "C:\\source\\file\\path.mkv" );
 
for ( let time of times ) {
    const [ v, a ] = trim( file.select( 'v' ), file.select( 'a' ), time[ 0 ], time[ 1 ] );
 
    video.push( v );
    audio.push( a );
}
 
 
// Note that each stream in this case needs to be a stream factory: a method that returns a stream
const [ sepV, sepA ] = [ () => color( 'black', 1920, 1080, 3 ), () => silence( 3 ) ];
 
// Adds a 3-second black scene between eahc original scene
const [ outV, outA ] = concat( separator( video, sepV ), separator( audio, sepA ) );
 
console.log( command( [ outV, outA ], 'C:\\generated\\file\\path.mkv', { outputArgs: [ '-f', 'matroska', '-y' ] } ).generate() );

```
I get this error:
```
t.js:28
console.log( command( [ outV, outA ], 'C:\\generated\\file\\path.mkv', { outputArgs: [ '-f', 'matroska', '-y' ] } ).generate() );
                                                                                                                    ^

TypeError: command(...).generate is not a function
    at Object.<anonymous> (t.js:28:117)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11
```

I managed to get around it like this:
```js
const cmd = command( [ outV, outA ], 'C:\\generated\\file\\path.mkv', { outputArgs: [ '-f', 'matroska', '-y' ] } );

const fragments = cmd.createFragments();
const compiler = compile(fragments)
const cmp = cmd.compile(compiler);

console.log('cmp', cmp);
```

But I'm not sure that this is the way to use the library.

**Comment by pedromsilvapt** [comment]
Hi! I'm glad this project was useful and sorry about the issues you encountered. I've made changes to the library in the past and forgot to update the example in README.

Yes, the way you did it works although some of the lines are not necessary. This way should work:
```typescript
const cmd = command( [ outV, outA ], 'C:\\generated\\file\\path.mkv', { outputArgs: [ '-f', 'matroska', '-y' ] } );

console.log( compile( cmd ).toString() );
```
Honestly, the API for this project can feel a bit contrived. I created it a long time ago and never had the energy to redo it. 😆 

So, if you encounter any other issues, feel free to ask!
